ui_testing/pages/testunit_page.py

```diff
@@ -249,3 +249,16 @@ def set_qualitative_value(self, value=''):
         qualitative_value = self.base_selenium.find_element_in_element(destination_element='general:input',
                                                                        source_element='test_unit:qualitative_value')
         qualitative_value.send_keys(value)
+
+    # if random is false, take the arguments that are not None only
+    # if random is true, take all the arguments, check if a value is given, otherwise generate random values
+    def update_testunit(self, number, name, iterations, category, random=True):
```

Comment:
```suggestion
    def update_testunit(self, number='', name='', iterations='', category='', random=True):
```

Reply:
I don't need to give them default value empty string, I want them to be null in case of having random = true, all the testunit's data will be updated by the given values in the method parameters, if a parameter is None, it should be randomly generated.

Reply:
![image](https://user-images.githubusercontent.com/17981089/71578055-da3d6700-2af6-11ea-96b6-84e9716b2ed0.png)

Reply:
Done